src/6_branch.js

```diff
@@ -272,6 +272,15 @@ Branch.prototype['init'] = wrap(
 
 		options = (options && typeof options === 'function') ? { } : options;
 
+		var branchViewId = null;
+
+		if (options) {
+			branchViewId = options.branch_view_id || null;
+		}
+		if (!branchViewId) {
+			branchViewId = utils.getParameterByName('_branch_view_id') || null;
```

Comment:
I did that intentionally, to reduce the chance of collision with partner QSPs.  It might not be needed though.